ci-operator/config/openshift/ovs-cni/openshift-ovs-cni-main.yaml

```diff
@@ -0,0 +1,32 @@
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+promotion:
+  to:
+  - name: "4.18"
```

Comment:
Hi please check the other sriov components that use a different name.
that way we are going to be able and pull them as part of the operator